passive/ShipCave.h

```diff
@@ -8,15 +8,17 @@
 class ShipCave : public FairModule
 {
   public:
+    ShipCave(Double_t z);
     ShipCave(const char* name, const char* Title="Exp Cave");
     ShipCave();
     virtual ~ShipCave();
     virtual void ConstructGeometry();
 
 
   private:
+    Double_t zEndOfProxShield;
     Double_t world[3];
-    ClassDef(ShipCave,1) //PNDCaveSD
+    ClassDef(ShipCave, 1) Int_t InitMedium(TString name);
```

Comment:
I'm not sure we need to declare this method, as it's part of FairModule, if I'm not mistaken.
We definitely need to fix the formatting.

Reply:
Yes